### ISSUE TITLE
pre 5.0 gone?

### ISSUE THREAD

**Issue by jrochkind** [railsdiff/railsdiff#12]
One of the things I love railsdiff the most for is when upgrading an app across a Rails major version boundary. 

But today I went to railsdiff.org, and it seems to only know about rails 5.0 and later?

I miss the earlier ones!  It was great to see how a default rails app had changed going way back. 

I am not sure if this is intentional or a bug?

**Comment by jrochkind** [comment]
I see everything going way back still seems avail at https://github.com/railsdiff/api/tree/master/generated

So maybe this is a bug?

**Comment by jrochkind** [comment]
Ah wait, and now the old ones are back for me on railsdiff.org. Temporary bug?

**Comment by jswanner** [comment]
The UI forces you to compare older source version to newer target version. So, if you set the source version as 5.0.0, then the target version options will be 5.0.1+. Perhaps you were just looking at the target options?

**Comment by jrochkind** [comment]
I swear I wasn't, I didn't have options before 5.0.0.beta1 available in 'source'....

Aha. Because I was uncharacteristically using Firefox. 

In Firefox, no options for source are given earlier than 5.0.0.beta1. 

In Chrome no problem. 

No idea how this would possibly be browser-dependent, but that's what I'm seeing!

**Comment by jswanner** [comment]
Oh, I see what you're saying. Firefox doesn't give much height to how it displays the `<select>`'s options dropdown, that said, all the options are there but you need to scroll that small dropdown.

<img width="169" alt="screen shot 2018-07-09 at 11 19 03" src="https://user-images.githubusercontent.com/261/42468481-eaec6578-8369-11e8-84d4-b8231487df9e.png">

**Comment by jrochkind** [comment]
Oh wow, mine cut off RIGHT below 5.0.0.beta1, with no half-line next, and what with that seeming like a reasonable boundary (5.x only), I got confused!

![screen shot 2018-06-19 at 1 21 16 pm](https://user-images.githubusercontent.com/149304/42471174-c4b480ae-838a-11e8-8905-bc16c2bafa56.png)

But okay, see what's going on now -- indeed it is scrollable. The bad UX is on Firefox's part I guess. Closing! Thanks!
